doc/Makefile

```
@@ -81,8 +81,8 @@ gitwash-update:
 #
 
 #SPHINXBUILD="LANG=C sphinx-build"
-NUMPYVER:=$(shell $(PYTHON) -c "import numpy; print(numpy.version.git_revision[:10])" 2>/dev/null)
-GITVER ?= $(shell cd ..; $(PYTHON) -c "import versioneer as v; print(v.get_versions()['full-revisionid'][:10])")
+NUMPYVER:=$(shell $(PYTHON) -c "import numpy; print(numpy.version.git_revision[:9])" 2>/dev/null)
+GITVER ?= $(shell cd ..; git rev-parse --short HEAD || echo Unknown)
```

We should be explicit about the length
```suggestion
GITVER ?= $(shell cd ..; git rev-parse --short=9 HEAD || echo Unknown)
```